src/config.ts

```diff
@@ -924,5 +924,5 @@ export const FF_ROUTING_PUSH_NOTIF_BETA_TESTER_SHA_LIST = pipe(
 export const FF_ROUTING_PUSH_NOTIF_CANARY_SHA_USERS_REGEX = pipe(
   process.env.FF_ROUTING_PUSH_NOTIF_BETA_TESTER_SHA_LIST,
```

Comment:
wht using FF_ROUTING_PUSH_NOTIF_BETA_TESTER_SHA_LIST here?

Reply:
My bad, wrong copy-paste :D Done in [123d52b](https://github.com/pagopa/io-backend/pull/960/commits/123d52bed584e45c391df90c13cb28c3bacf43e7)